core/workspace_comment/workspace_comment_render_svg.js

```diff
@@ -30,62 +30,73 @@ goog.require('Blockly.WorkspaceCommentSvg');
 
 
 /**
- * Width of the border around the text area.
+ * Size of the resize icon.
  * @type {number}
  * @const
  */
-Blockly.WorkspaceCommentSvg.BORDER_WIDTH = 10;
+Blockly.WorkspaceCommentSvg.RESIZE_SIZE = 8;
+
+/**
+ * Radius of the border around the comment.
+ * @type {number}
+ * @const
+ */
+Blockly.WorkspaceCommentSvg.BORDER_RADIUS = 3;
 
 /**
  * Offset from the foreignobject edge to the textarea edge.
  * @type {number}
  * @const
  */
-Blockly.WorkspaceCommentSvg.TEXTAREA_OFFSET = 4;
+Blockly.WorkspaceCommentSvg.TEXTAREA_OFFSET = 2;
+
+/**
+ * Offset from the top to make room for a top bar.
+ * @type {number}
+ * @const
+ */
+Blockly.WorkspaceCommentSvg.TOP_OFFSET = 10;
+
+/**
+ * Returns a bounding box describing the dimensions of this comment.
+ * @return {!{height: number, width: number}} Object with height and width
+ *    properties in workspace units.
+ */
+Blockly.WorkspaceCommentSvg.prototype.getHeightWidth = function() {
+  return { width: this.getWidth(), height: this.getHeight() };
+};
 
 Blockly.WorkspaceCommentSvg.prototype.render = function() {
-  this.rendered_ = true;
+  if (this.rendered_) return;
```

Comment:
```
if (this.rendered_) {
  return;
}
```

The style guide [says](https://google.github.io/styleguide/jsguide.html#formatting-braces-all) that this is the one case where you can do it inline, but the blockly codebase avoids it.